extension/src/cli/dvc/contract.ts

```diff
@@ -62,7 +62,6 @@ export const fileHasError = (file: FileDataOrError): file is DvcError =>
 
 export const DEFAULT_EXP_SHOW_OUTPUT = [
   {
-    branch: undefined,
```

Comment:
[F] This is only added when sending data to the webview. No need for it to be in the output.